test/timeline.js

```diff
@@ -0,0 +1,203 @@
+const { expect } = require('chai')
+const nockHelper = require('./helpers/nock_helper')
+const Hubspot = require('..')
+
+const userId = process.env.USER_ID || 23456
+const applicationId = process.env.APPLICATION_ID || 12345
+
+describe('timeline', function() {
+  const hubspot = new Hubspot({
+    accessToken: process.env.ACCESS_TOKEN || 'some-fake-token',
```

Comment:
Same question as on #131, what happens to the test if the env variable is not set?

Reply:
Good question! As I've outline [here](https://github.com/MadKudu/node-hubspot/pull/132#discussion_r241098575) the test will pass without this env variable.